### ISSUE TITLE
Word Glossary I have written

### ISSUE THREAD

**Issue by bob-g4bby** [zeroflag/punyforth#60]
Attila,
I've written a glossary for punyforth in LibreOffice. I couldn't work out what a few of the words do, but it is about 99% done. You are welcome to include it with the code.
regards, Bob SW U.K. bob.edwards50@yahoo.com
[Glossary of Punyforth Words.docx](https://github.com/zeroflag/punyforth/files/5415857/Glossary.of.Punyforth.Words.docx)
[Glossary of Punyforth Words.pdf](https://github.com/zeroflag/punyforth/files/5415858/Glossary.of.Punyforth.Words.pdf)

**Comment by zeroflag** [comment]
This is awesome, thanks for your work, I'm going to add it to to the repo.

**Comment by bob-g4bby** [comment]
Hi Attila,
You're welcome, thank you for publishing Punyforth. It's been fun running (and crashing) small programs, to learn how stuff works. The glossary has all the words included but some descriptions are missing through lack of information. There is a small list of unsorted words you might like to place and descriptions as you like.
To talk to ESP8266 modules via UDP etc, I use the free community edition of Labview, a very powerful test and measurement ide I used for many years before I retired. Once you know it, like forth it saves a lot of time over conventional languages and runs fast. You get something running very quickly!
    On Wednesday, 21 October 2020, 18:44:00 BST, Attila Magyar <notifications@github.com> wrote:  
 
 


This is awesome, thanks for your work, I'm going to add it to to the repo.

**Comment by bob-g4bby** [comment]
[https://www.ni.com/en-gb/shop/labview/select-edition/labview-community-edition.html](url)

**Comment by bob-g4bby** [comment]
Hi Attila,

You're welcome, thank you for publishing Punyforth. It's been fun running (and crashing) small programs, to learn how stuff works. The glossary has all the words included but some descriptions are missing through lack of information. There is a small list of unsorted words you might like to place and descriptions as you like.

To talk to ESP8266 modules via UDP etc, instead of Python I use the [LabView Community Edition](https://www.ni.com/en-gb/shop/labview/select-edition/labview-community-edition.html) a very powerful test and measurement language for Windows I used for many years before I retired. Once you know it, like forth it saves a lot of time over conventional languages and runs fast. You get something running very quickly!
